mobile/ios/scripts/build_frameworks.sh

```diff
@@ -1,53 +1,59 @@
 #!/bin/sh -e
 
-rm -rf Frameworks/*.framework
-mkdir Frameworks/boost.framework
-mkdir Frameworks/event2.framework
-mkdir Frameworks/event2_pthreads.framework
-mkdir Frameworks/measurement_kit.framework
-mkdir Frameworks/yaml-cpp.framework
-
-# XXX Copying x86 headers -- has this implications (e.g., config.h)?
-for N in boost event2 measurement_kit yaml-cpp; do
-    cp -Rp build/iPhoneSimulator/i386/include/$N Frameworks/$N.framework/Headers
-done
-
-lipo -create -output Frameworks/boost.framework/boost \
-  build/iPhoneOS/arm64/lib/libboost.a \
-  build/iPhoneOS/armv7s/lib/libboost.a \
-  build/iPhoneOS/armv7/lib/libboost.a \
-  build/iPhoneSimulator/i386/lib/libboost.a \
-  build/iPhoneSimulator/x86_64/lib/libboost.a
-
-lipo -create -output Frameworks/event2.framework/event2 \
-  build/iPhoneOS/arm64/lib/libevent.a \
-  build/iPhoneOS/armv7s/lib/libevent.a \
-  build/iPhoneOS/armv7/lib/libevent.a \
-  build/iPhoneSimulator/i386/lib/libevent.a \
-  build/iPhoneSimulator/x86_64/lib/libevent.a
-
-lipo -create -output Frameworks/event2_pthreads.framework/event2_pthreads \
-  build/iPhoneOS/arm64/lib/libevent_pthreads.a \
-  build/iPhoneOS/armv7s/lib/libevent_pthreads.a \
-  build/iPhoneOS/armv7/lib/libevent_pthreads.a \
-  build/iPhoneSimulator/i386/lib/libevent_pthreads.a \
-  build/iPhoneSimulator/x86_64/lib/libevent_pthreads.a
-
-lipo -create -output Frameworks/measurement_kit.framework/measurement_kit \
-  build/iPhoneOS/arm64/lib/libmeasurement_kit.a \
-  build/iPhoneOS/armv7s/lib/libmeasurement_kit.a \
-  build/iPhoneOS/armv7/lib/libmeasurement_kit.a \
-  build/iPhoneSimulator/i386/lib/libmeasurement_kit.a \
-  build/iPhoneSimulator/x86_64/lib/libmeasurement_kit.a
-
-lipo -create -output Frameworks/yaml-cpp.framework/yaml-cpp \
-  build/iPhoneOS/arm64/lib/libyaml-cpp.a \
-  build/iPhoneOS/armv7s/lib/libyaml-cpp.a \
-  build/iPhoneOS/armv7/lib/libyaml-cpp.a \
-  build/iPhoneSimulator/i386/lib/libyaml-cpp.a \
-  build/iPhoneSimulator/x86_64/lib/libyaml-cpp.a
-
-# Create fake header for event2_pthreads
-install -d Frameworks/event2_pthreads.framework/Headers
-echo "/* Make CocoaPod happy */" > \
-  Frameworks/event2_pthreads.framework/Headers/event2_pthreads_dummy.h
+ROOTDIR=$(cd $(dirname $0) && pwd -P)
+
+(
+    cd "$ROOTDIR/.."
+ 
+    rm -rf Frameworks/*.framework
+    mkdir Frameworks/boost.framework
+    mkdir Frameworks/event2.framework
+    mkdir Frameworks/event2_pthreads.framework
+    mkdir Frameworks/measurement_kit.framework
+    mkdir Frameworks/yaml-cpp.framework
+
+    # XXX Copying x86 headers -- has this implications (e.g., config.h)?
```

Comment:
For the records. This is the wrong thing to do as long as libevent is involved. In fact, libevent has architecture specific headers. Hence, here we were installing some headers with the wrong ABI.

Luckily, from now on we don't need anymore to install libevent headers because only measurement-kit headers are needed to use measurement-kit.